students/tony7kh/hw-8/task-2.js

```diff
@@ -0,0 +1,9 @@
+const createArraysWithObjects = (...names) => {
+    let resultArray = []
+    names.map((item, index) => {
+        resultArray.push({ 'name': item, 'id': ++index })
+    })
```

Comment:
`Array.map` returns new array with modified elements of the initial one [info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map?retiredLocale=uk). 

```suggestion
    const resultArray = names.map((item, index) => ({ 'name': item, 'id': ++index }));
```